lib/smack/Naming.cpp

```diff
@@ -85,6 +85,8 @@ const std::string Naming::CONTRACT_EXPR = "$expr";
 const std::string Naming::MEMORY_SAFETY_FUNCTION =
     "__SMACK_check_memory_safety";
 const std::string Naming::MEMORY_LEAK_FUNCTION = "__SMACK_check_memory_leak";
+const std::pair<std::string, std::string> Naming::INT_WRAP_FUNCTIONS = {"$tos",
```

Comment:
Maybe this should be a map from bool to string? Then you don't have to introduce the getter function.

Reply:
I think I prefer the existing implementation since having a getter function increases readability. If it were implemented using a map, getting a function will be something like `Naming::INT_WRAP_FUNCTIONS[true]` which is less meaningful than `Naming::getIntWrapFunc(true)`.

Reply:
Ok, sounds good.